sql/core/src/test/scala/org/apache/spark/sql/test/DataFrameReaderWriterSuite.scala

```diff
@@ -56,10 +62,6 @@ class DefaultSource
   with SchemaRelationProvider
   with CreatableRelationProvider {
 
-  case class FakeRelation(sqlContext: SQLContext) extends BaseRelation {
```

Comment:
Move `FakeRelation` to `DataSourceTest`. then `FakeDataSourceStrategy` can use it.